layouts/partials/meta.html

```diff
@@ -9,6 +9,9 @@
 {{ $locale      := site.Params.locale }}
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta
+  name="description"
+  content="KEDA is a Kubernetes-based Event Driven Autoscaler. With KEDA, you can drive the scaling of any container in Kubernetes based on the number of events needing to be processed.">
```

Comment:
Good of you to point out that pages need a `<meta name="description">`, but this isn't the way to do it.
Revert this change.

Reply:
This will be tracked via:

- #894